testing/tests/DevExpress.viz.gauges/barGauge.tests.js

```diff
@@ -574,6 +575,104 @@ QUnit.test('Some values are not changed', function(assert) {
     }, this);
 });
 
+QUnit.test('label location if the angle is < 90 and resolveLabelOverlapping - shift', function(assert) {
+    const done = assert.async();
+    this.$container.dxBarGauge({
+        values: [10, 11, 12],
+        resolveLabelOverlapping: 'shift'
+    });
+    const group = this.getBarsGroup();
+
+    group.animationComplete = $.proxy(function() {
+        assert.deepEqual(group.children[2]._stored_settings.points, [64.04786, 219.49845, 21, 7], 'line 1 coords');
+        assert.equal(group.children[3]._stored_settings.x.toFixed(3), 34.027, 'text 1 is coord x');
```

Comment:
Please, let's use `strictEqual` everywhere

Reply:
Fixed for stricter verification